src/core/security/credentials.c

```diff
@@ -179,8 +179,8 @@ void grpc_server_credentials_set_auth_metadata_processor(
   GRPC_API_TRACE(
       "grpc_server_credentials_set_auth_metadata_processor("
       "creds=%p, "
-      "processor=grpc_auth_metadata_processor { process: %lx, state: %p })",
-      3, (creds, (unsigned long)processor.process, processor.state));
```

Comment:
to me it seems like there's no guarantee that  long long will be of same size as gpr_uint64.
Given the formatting string is %llx, I should really use a long long type as an argument.

Actually, based on this post http://stackoverflow.com/questions/14071713/what-is-wrong-with-printfllx, it seems like the %llx might not work on windows :-)

I genuinely don't know what the correct approach is.
Btw, I did some %ll  (long long)  prints in some of my previous pull requests.